EIPS/eip-1328.md

```diff
@@ -0,0 +1,49 @@
+---
+eip: 1328
+title: WalletConnect Standard URI Format
+authors: ligi <ligi@ligi.de>, Pedro Gomes <pedrouid@protonmail.com>
+type: Standards Track
+category: ERC
+status: Draft
+created: 2018-08-15
+discussions-to: https://ethereum-magicians.org/t/wallet-connect-eip/850
+requires: 831
+---
+
+## Simple Summary
+
+A standard way of creating WalletConnect URIs for establishing connections between wallets and dapps
+
+## Abstract
+
+Sesion data in QR codes for initiating a connection between a wallet and dapp using the WalletConnect standard require a standardized URI format for effictively parsing the intenet. This ERC is expandable between versions of the standard and also mobile-to-mobile deep links.
```

Comment:
intenet needs to be changed as it is not a word ;-)

In general I would change it to:

This standard defines how the data to connect some application and a wallet can be encoded with a URI. This URI can then be shown either as a QR code or for mobile to mobile as a link.